Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -182,6 +183,32 @@ ThrowCompletionOr<void> NewArray::execute_impl(Bytecode::Interpreter& interprete
     return {};
 }
 
+ThrowCompletionOr<void> ConcatArray::execute_impl(Bytecode::Interpreter& interpreter) const
+{
+    auto& vm = interpreter.vm();
+
+    auto& lhs = interpreter.reg(m_lhs).as_array();
+    size_t lhs_length = TRY(length_of_array_like(vm, lhs));
```

Comment:
Again, observable by calling into user code. Add spec comments or it's not gonna happen.